src/UKHO.ERPFacade.API/appsettings.json

```diff
@@ -38,7 +38,11 @@
         "BaseAddress": "",
         "SapServiceOperation": "Z_ADDS_MAT_INFO",
         "Username": "",
-        "Password": ""
+        "Password": "",
+        "RosBaseAddress": "",
```

Comment:
Could we name this so that its intent is clear - "ros" doesn't mean anything to a number of people that will need to read this code in future.

Reply:
Updated the configuration names.